src/functions/Funcs/user/userDefaultAvatar.js

```diff
@@ -1,7 +1,7 @@
 module.exports = async d => {
     const data = d.util.aoiFunc(d);
 
-    const [userID = d.author?.id, size = 4096, dynamic = 'true', format] = data.inside.splits;
+    const [userID = d.author?.id, size = 4096, dynamic = 'true', extension] = data.inside.splits;
```

Comment:
defaultAvatarUrl doesnt support properties so we can remove image options and just keep userid